### ISSUE TITLE
list all Gridcoin media at 1 place

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#149]
there is:

* new [logo packs](https://steemit.com/gridcoin/@joshoeah/gridcoin-logo-packs-get-em-while-they-re-hot)

<br>

* [ads, images and banners](https://cryptocurrencytalk.com/topic/50138-gridcoin-ads-images-and-banners/)
* [images on website](https://github.com/grctest/Gridcoin-Site/tree/master/source/Images)
* contest: [create Gridcoin media](https://steemit.com/gridcoin/@erkan/1million-block-event-with-prize-create-gridcoin-media-pictures-videos-songs) (pictures, videos, songs, ...)
* [What do you think about this new picure?](https://steemit.com/gridcoin/@erkan/what-do-you-think-about-this-new-gridcoin-picture) (incl. branding, new logo)

see also task:
* [list + renew videos about Gridcoin (out-of-date)](https://github.com/Erkan-Yilmaz/Gridcoin-tasks/issues/114)

**Comment by MagixInTheAir** [comment]
Any media with the new logo ?

**Comment by Erkan-Yilmaz** [comment]
seen the "logo packs" link above?

**Comment by MagixInTheAir** [comment]
Yes. Images with the old logo should be marked as outdated, and we should involve the community into making banners and ads with the new one :) sadly I don't have the skills

**Comment by Erkan-Yilmaz** [comment]
ok, I have added the link on top now
